src/linter/report.go

```diff
@@ -557,6 +557,19 @@ $foo = new Foo();`,
 			Before:   `sprintf("id=%d")`,
 			After:    `sprintf("id=%d", $id)`,
 		},
+
+		{
+			Name:     "invalidClassName",
```

Comment:
Please add the diagnostic in a separate PR.
This PR should be about fixing the referenced issue.